.github/workflows/test.yml

```diff
@@ -347,6 +347,48 @@ jobs:
           min_coverage: 67
           token: ${{ github.token }}
   
+  ##############################################################################
+  # JOB: UNIT TEST BACKEND  ####################################################
+  ##############################################################################
+  unit_test_backend:
+    name: Unit tests - Backend
+    runs-on: ubuntu-latest
+    needs: [build_test_backend]
+    steps:
+      ##########################################################################
+      # CHECKOUT CODE ##########################################################
+      ##########################################################################
+      - name: Checkout code
+        uses: actions/checkout@v2
+      ##########################################################################
+      # DOWNLOAD DOCKER IMAGES #################################################
+      ##########################################################################
+      - name: Download Docker Image (Backend)
+        uses: actions/download-artifact@v2
+        with:
+          name: docker-backend-test
+          path: /tmp
+      - name: Load Docker Image
+        run: docker load < /tmp/backend.tar
+      ##########################################################################
+      # UNIT TESTS BACKEND #####################################################
+      ##########################################################################
+      - name: backend | Unit tests
+        run: |
+          docker run -v ~/coverage:/app/coverage --rm gradido/backend:test yarn run test
+          cp -r ~/coverage ./coverage
```

Comment:
This means running without database, correct?

Reply:
Until now, yes